### PR TITLE
Add team summary endpoint

### PR DESCRIPTION
**Pull request by djmarsu**

### CODE REVIEW

File: app/controllers/helpdesk_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class HelpdeskController < ApplicationController
+  before_action :init
+
+  def init
+    time = Time.zone.now
+    @start_time = time.beginning_of_day
+    @end_time = time.end_of_day
+    @team = Team.find_by_name('Helpdesk')
+    @contacts_service = ContactsService.new(@team, @start_time, @end_time)
+    @agent_statuses = AgentStatus.joins(:agent).where(open: true, agents: { team_id: Team.find_by_name('Helpdesk').id })
```
Comment (siiptuo) [inline app/controllers/helpdesk_controller.rb]: team already in variable

File: app/controllers/helpdesk_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  def queuers_count
+    @contacts_service.queue_contacts.length
+  end
+
+  def index
+    render json: {
+      agents_online_all: @agent_statuses.length,
+      agents_online_free: free_agents.length,
+      queue_count: queuers_count,
```
Comment (siiptuo) [inline app/controllers/helpdesk_controller.rb]: better naming: `queuer_count` or `queue_length`

File: app/models/helpdesk.rb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+class Helpdesk < ActiveRecord::Base
+  belongs_to :agent
+end
```
Comment (siiptuo) [inline app/models/helpdesk.rb]: not used?

**Comment by siiptuo** [comment]
whole thing could be named to better such as "helpdesk summary"

File: app/controllers/helpdesk_controller.rb (hunk 1 above)
Comment (djmarsu) [inline app/controllers/helpdesk_controller.rb]: oho joo

**Pull request merged**